src/test/java/org/dz/solution/task32/Task32Test.java

```diff
@@ -0,0 +1,60 @@
+package org.dz.solution.task32;
+
+import org.junit.Test;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+
+public class Task32Test {
+    int[] array;
+
+    @Test
+    public void test() throws IOException {
+        String sourceDataFile = "src/main/resources/org/dz/solution/task32/InputFile";
+        String resultFile = "src/main/resources/org/dz/solution/task32/OutputFile";
+        Files.delete(Path.of("src/main/resources/org/dz/solution/task32/OutputFile"));
+
+        PrintWriter pw = new PrintWriter(resultFile);
+        readingAFile(sourceDataFile);
+        pw.print(Arrays.toString(findTheMissingNumbers(array)));
```

Comment:
Path the number of numbers in the file instead of array. Array has to be created inside  findTheMissingNumbers